### PR TITLE
W-16312987-alt-text-trust-day-1.11-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 1]
```diff
@@ -27,11 +27,11 @@ Anypoint Runtime Fabric on VMs / Bare Metal supports development and production
 
 The development configuration is intended for testing only. It requires at least one controller and two worker nodes. The controller node runs the internal load balancer and agents used to connect to Anypoint Platform. Communication between the agent and Anypoint Platform is always outbound. Multiple replicas of application can run across workers.
 
-image::architecture-development.png[]
+image::architecture-development.png["Diagram of a network development architecture of MuleSoft control plane with controllers, such as internal load balancer, agent, and workers"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as an internal load balancer, agent, and workers"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 2]
```diff
@@ -27,11 +27,11 @@ Anypoint Runtime Fabric on VMs / Bare Metal supports development and production
...
 
 === Production Configuration
 
-image::architecture-production.png[]
+image::architecture-production.png["Diagram of a Mule development with control plane, controllers, such as internal load balancer, agent, workers, and TCP load balancer"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-production.png["Diagram of a Mule development with the control plane, controllers, such as an internal load balancer, agent, workers, and TCP load balancer"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 3]
```diff
@@ -45,7 +45,7 @@ Mule applications run on workers. Multiple replicas of applications can run acro
 
 The following diagram shows the general network architecture of Runtime Fabric on VMs / Bare Metal.
 
-image::architecture-network.png[]
+image::architecture-network.png["Diagram of network architecture of Runtime Fabric and TCP load balancer connection with internal load balancers of controllers and workers"]
 
 This diagram shows the TCP load balancer used to load balance requests across the internal load balancers running on the controllers. It also shows the internal load balancer that distributes requests to each replica of Mule applications running on the workers.
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc:50]: When you describe the image in the surrounding text (best practice!), you don't need to repeat in alt text. In this case, if you have described every element of the diagram already in the surrounding text, then you can [""]. If there's something that's not mentioned in the surrounding text, put only that.

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 4]
```diff
@@ -84,7 +84,7 @@ The successful operation of Anypoint Runtime Fabric on VMs / Bare Metal is a sha
 
 The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
 
-image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsiblities]
+image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsibilities]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]: If decorative, [""]

File: modules/ROOT/pages/install-create-rtf-arm.adoc [hunk 5]
```diff
@@ -25,7 +25,7 @@ If you do not see the *Runtime Fabrics* tab in Runtime Manager as described belo
 [IMPORTANT]
 After creating the Runtime Fabric, Runtime Manager displays the corresponding activation data. Record this data to use later when installing running the Runtime Fabric on VMs / Bare Metal installer.
 
-image::runtime-fabric-arm.png[]
+image::runtime-fabric-arm.png["Runtime Fabric instance installation steps in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-create-rtf-arm.adoc]:
```suggestion
image::runtime-fabric-arm.png["Runtime Fabric instance installation steps in the Runtime Manager UI"]
```

File: modules/ROOT/pages/install-prereqs.adoc [hunk 6]
```diff
@@ -413,7 +413,7 @@ server pool consisting of each of the controller nodes is sufficient.
 Runtime Fabric on VMs / Bare Metal requires a minimum of three nodes in a development environment as shown in the
 following diagram:
 
-image::architecture-development.png[]
+image::architecture-development.png["Diagram of a network development architecture of MuleSoft control plane with controllers, such as internal load balancer, agent, and workers"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-prereqs.adoc]:
```suggestion
image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as an internal load balancer, agent, and workers"]
```

File: modules/ROOT/pages/install-prereqs.adoc [hunk 7]
```diff
@@ -323,7 +323,7 @@ The controller VM that is the leader for installation performs the following act
 A minimum of 6 nodes are required for a production configuration of Runtime Fabric on VMs/ Bare Metal, as shown in
 the following diagram.
 
-image::architecture-production.png[]
+image::architecture-production.png["Diagram of a Mule development with control plane, controllers, such as internal load balancer, agent, workers, and TCP load balancer"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-prereqs.adoc]:
```suggestion
image::architecture-production.png["Diagram of a Mule development with control plane, controllers, such as an internal load balancer, agent, workers, and TCP load balancer"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 8]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
-image::app-versioning-config-applied-success.png[]
+image::app-versioning-config-applied-success.png["Configuration applied successfully for a Mule app in Runtime Manager UI"]
 +
 The application status, configuration identifier, date, and other information is updated in the status box.  The new configuration is displayed at the top of the *Config changes* list, along with a *Last successful* label to indicate that it's the last configuration successfully deployed.
 +
 If the updates to the configuration fail to deploy, you get an `Desired configuration wasn't applied` message, and the *Configuration status* displays an `Update failed` label. The configuration is displayed in the *Config changes* history without a label.
 +
-image::app-versioning-update-failed.png[]
+image::app-versioning-update-failed.png["Configuration failed to apply for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-update-failed.png["Configuration failed to apply status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 9]
```diff
@@ -35,7 +35,7 @@ In the *Config changes* tab on the application's *Settings* page. Runtime Fabric
 +
 The last ten configuration deployments are displayed in order of their respective configuration date. Each configuration in the history has a unique hash to identify it.
 
-image::app-versioning.png[]
+image::app-versioning.png["Configuration page for a runtime fabric application, displaying application file, version, deployment target, runtime version, and replicas"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning.png["Configuration page for a Runtime Fabric application, displaying the application file, version, deployment target, runtime version, and replicas"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 10]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Replicas Status for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-replica-status.png["Replicas status for a Mule app in Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 11]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
 Once the configuration is successfully applied and deployed, a message appears at the top of the page.
 +
-image::app-versioning-config-applied-success.png[]
+image::app-versioning-config-applied-success.png["Configuration applied successfully for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-config-applied-success.png["Configuration applied successfully status for a Mule app in Runtime Manager UI"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc (hunk 4 above)
Comment (dmerlob) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::runtimefabricresponsibility.png["Runtime Fabric Shared Responsibilities"]
```

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**